pyxem/tests/utils/test_deprecation.py

```diff
@@ -0,0 +1,115 @@
+import warnings
```

Comment:
Most, if not all other files, have the license header, right? I suggest to add it here as well, for consistency.